cpp/test/gpu_operations_test/matrix_scalar_add_test.cc

```diff
@@ -25,15 +25,32 @@
 #include "Eigen/Dense"
 #include "gtest/gtest.h"
 
-TEST(GPU_MATRIX_SCALAR_ADD, Basic_Test) {
-  Nice::Matrix<float> a(3,4);
-  a << 1, 2, 3, 4,
-       1, 2, 3, 4,
-       1, 2, 3, 4;
-  Nice::Matrix<float> b = Nice::GpuOperations<float>::Add(a, 1);
-  Nice::Matrix<float> answer(3, 4);
-  answer << 2, 3, 4, 5,
-            2, 3, 4, 5,
-            2, 3, 4, 5;
-  ASSERT_TRUE(answer.isApprox(b));
+template<class T>
+class GPU_MATRIX_SCALAR_ADD : public ::testing::Test {
```

Comment:
Could you change the class name to camel case one like GpuMatrixScalarAddTest? Thanks a lot!